--- a/package/kernel/linux/modules/netsupport.mk
+++ b/package/kernel/linux/modules/netsupport.mk
@@ -370,6 +370,16 @@
 endef
 
 
+define KernelPackage/psample
+  SUBMENU:=Network Support
+  TITLE:=psample: netlink sampling module
+  DEPENDS:=
+  KCONFIG:= \
+       CONFIG_PSAMPLE
+  FILES:=$(LINUX_DIR)/net/psample/psample.ko
+  AUTOLOAD:=$(call AutoLoad,60,psample)
+endef
+$(eval $(call KernelPackage,psample))
