--- ./package/base-files/files/bin/config_generate
+++ ./package/base-files/files/bin/config_generate
@@ -147,8 +147,8 @@ generate_network() {
		static)
			local ipad
			case "$1" in
-				lan) ipad=${ipaddr:-"192.168.1.1"} ;;
-				*) ipad=${ipaddr:-"192.168.$((addr_offset++)).1"} ;;
+				lan) ipad=${ipaddr:-"10.10.10.100"} ;;
+				*) ipad=${ipaddr:-"10.10.$((addr_offset++)).1"} ;;
			esac

			netm=${netmask:-"255.255.255.0"}
