--- feeds.conf.default	2025-08-11 21:50:11.331258500 +0800
+++ 1feeds.conf.default	2025-08-11 21:51:35.028355200 +0800
@@ -1,6 +1,6 @@
 src-git packages https://github.com/coolsnowwolf/packages
 #src-git luci https://github.com/coolsnowwolf/luci
-src-git luci https://github.com/coolsnowwolf/luci.git;openwrt-23.05
+src-git luci https://github.com/coolsnowwolf/luci.git;openwrt-24.10
 src-git routing https://github.com/coolsnowwolf/routing
 src-git telephony https://github.com/coolsnowwolf/telephony.git
 #src-git helloworld https://github.com/fw876/helloworld.git
@@ -8,4 +8,4 @@
 #src-git video https://github.com/openwrt/video.git
 #src-git targets https://github.com/openwrt/targets.git
 #src-git oldpackages http://git.openwrt.org/packages.git
-#src-link custom /usr/src/openwrt/custom-feed
+#src-link custom /usr/src/openwrt/custom-feed
\ No newline at end of file
